README.md

```
@@ -60,274 +61,315 @@ Proovy의 도메인 모델은 **채팅 메시지, 수식 메타데이터, API 
-// ❌ 나쁜 예: 모호한 이름
-interface Props {
-  loading: boolean;
-  callback: () => void;
-  count: number;
-}
-```
+**1️⃣ 파일 생성 규칙 (File Naming)**
+
+| **파일 종류**         | **규칙 (Case)**        | **예시 (Example)** | **비고** |
+| --------------------- | ---------------------- | ------------------ | -------- |
+| **페이지 & 컴포넌트** | **PascalCase**         | `LoginPage.tsx`    |
```

markdownlint-cli2 (0.20.0)

[warning] Table column count
Expected: 4; Actual: 3; Too few cells, row will be missing data

(MD056, table-column-count)